Package.swift

```diff
@@ -5,9 +5,15 @@ import Foundation
 
 let registry = SDKRegistry()
 let version = "48.0.3"
-let mapboxCommonVersion = Version("11.0.1") 
+let mapboxCommonVersion = Version("11.0.1")
 let checksum = "9233bcb28551ded20157c22e78d5bdc0ff859e1e6423bed34461b3310f23cc22"
 
+func mapboxCommonVersionUpperBound(_ mapboxCommonVersion: Version) {
+    var version = mapboxCommonVersion
+    version.patch = 99999
+    return version
+}
```

Comment:
Can we have more generic function? something similar to:

```suggestion
func versionRangeThatAllowsOnlyPatches(for version: Version) -> Range<Version> {
    version..<Version("\(version.major).\(version.minor + 1).0")
}
```
 
Then we can get rid of comment below and use it as:
 ```swift
 .package(url: "https://github.com/mapbox/mapbox-common-ios.git", versionRangeThatAllowsOnlyPatches(for: mapboxCommonVersion)),
```